deploy/example/example-app.yaml

```diff
@@ -67,3 +67,6 @@ spec:
                   csi.cert-manager.io/issuer-name: ca-issuer
                   csi.cert-manager.io/dns-names: "${POD_NAME}.${POD_NAMESPACE}.svc.cluster.local"
                   csi.cert-manager.io/uri-sans: "spiffe://cluster.local/ns/${POD_NAMESPACE}/pod/${POD_NAME}}/${POD_UID}"
+                  csi.cert-manager.io/keystore-pkcs12-enable: "true"
+                  csi.cert-manager.io/keystore-pkcs12-password: "my-password"
+                  csi.cert-manager.io/keystore-pkcs12-file: "crt.p12"
```

Comment:
Would this read better as `pkcs12-keystore-file`, `pkcs12-keystore-password` instead?

I guess the reason for doing it this way is to make `keystore-jks` sort nearby to the pkcs12 keystore details? IMO that seems like low reward for making our attribute names read weirdly

Reply:
(allowing users to create an arbitrarily deep nested folder structure is also an attack vector of sorts, so we should limit the filename size and also ensure users can't create additional subdirectories)

Reply:
I agree with pkcs-password, since we will use the same password for key store and trust store. But a user should be able to set the filename of the trust store also, correct?

Reply:
Well, if you plan to implement that also, I am sold! 👍🙏❤️

Reply:
Please see https://github.com/cert-manager/csi-driver/pull/104

Reply:
`pkcs12-file` (or maybe `pkcs12-filename`?) & `pkcs12-password` sounds good to me.